gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -232,7 +232,7 @@ protected void createButtonsForButtonBar(Composite parent) {
       createButton(parent, IDialogConstants.CANCEL_ID, IDialogConstants.CANCEL_LABEL, false);
 
       Listener modifyListener = e -> {
-        ok.setEnabled(traceInput.isReady());
+        ok.setEnabled(traceInput.isReady() && traceInput.isDeviceValidated());
```

Comment:
Make `isReady` return false, rather than expose the `isDeviceValidated` method

Reply:
Putting it in `isReady` would cause us to display the `requiredFieldMessage` which would confuse users IMO